<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleQueryWidget</class>
 <widget class="QWidget" name="ScheduleQueryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="qeEngineerLineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>电器工程师</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qcEngineerLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>产品工程师</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qxCodeLineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>协众项目代号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qcCodeLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>客户项目代号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deEngineerLineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>电器工程师</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dcEngineerLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>产品工程师</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>项目开发进度查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>项目延期情况查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dcCodeLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>客户项目代号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dxCodeLineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>协众项目代号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qeBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qcBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dcBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
